### ISSUE TITLE
Site redesign

### ISSUE THREAD

**Issue by Klortho** [JATS4R/jats4r.github.io#23]
UPDATE:  See this [site redesign wiki page](https://github.com/JATS4R/JATS4R-Participant-Hub/wiki/Site-redesign) for this sub-project's info.

---

Our site leaves a lot to be desired.  I happened on a site today that I like a lot: [http://todogroup.org](http://todogroup.org), and I'd like to copy it, with style tweaks here and there as needed.

**Comment by seligym** [comment]
Hi Chris
I am very much in agreement with you and the site you gave an example of is
very nice and modern-looking.

I have a lot of experience with developing and redeveloping websites.
Usually I gather the business/functional requirements, help work on content
strategy, carry out the information architecture research, testing, and
plan, and work with the designers and developers to ensure functional
requirements are met etc. But I've also been known to do front-end work
when required. So I would love to help out with this.

Based on experience, I would strongly recommend that before you begin the
work of copying another site, the group might consider thinking about what
business and user goals the site should actually serve, and based on the
goals, arrange content, design tweaks, etc, to serve the goals. That way
we're not just making stuff fit in the new place. Defining goals will
enable us to (i) choose/arrange content and features that all work to serve
both JATS4R and the people who might visit the site (rather than just
throwing it up there) and (ii) provide a way for us to measure the success
of the site and refine it later on if needed. It's kind of like when you
move a house: you don't want to take stuff you don't really need or want to
the new place, and you want to plan ahead how you will use the space in the
new house and furnish accordingly. Another way of looking at is that you
wouldn't build a new house without a blueprint.

So even if we use your nice site's look and feel, we should still arrange
the content, information flow, functionality, etc. according to what JATS4R
requires, instead of just copying it over with no plan. For example, if one
of JATS4R's goals is to help raise the level of XML literacy (and JATS
literacy in particular), then we should determine how best to do that on
the new site (blog? static articles?), and plan accordingly. And if our
goal is to persuade users to support/join/adopt JATS4R, then should the
business reasons (or some catchy version thereof) be part of the site, and
if so, where?

One final note is about metrics. Site business goals should be measurable
using analytics tools (like Google Analytics), so that we can tell whether
the site is successful or not.

I was going to add the business requirements to the current site, but now
I'm wondering if I should wait until the group decides whether to just move
everything over as-is or think about some changes.

I hope these thoughts haven't been overwhelming. If you decide a
sub-committee would be best for working on this, I would be more than happy
to join. In any case, I love your idea to renovate the site.

Mary

P.S. I just noticed that you have a Google Analytics acct set up for the
site. This is something I use regularly to analyse my organization's web
properties to answer various business questions; would it be possible to
add me as an admin so I can take a look? Even apart from a site redev, this
will help to measure outreach campaign success and other goals.

**Comment by Klortho** [comment]
That's a lot! I agree with everything you wrote, but I'm also wondering how we can make sure that this stays agile, and that we don't get too bogged down in process.

As for the analytics, I'm afraid that I just [copied that over from the eregs site](https://github.com/JATS4R/jats4r.github.io/commit/1b94c4c44837b907c83865e7bb0a817cf7f6ae29#diff-eacf331f0ffc35d4b482f1d15a887d3bR228) and never looked at it. So, it still needs to be set up.  If you could do it, that would be great -- I'll make a new issue.

**Comment by Klortho** [comment]
I'd think we can use the wiki here to come up with requirements, and use the issues to discuss implementation. How does that sound?

[Edit: I went ahead and created the [Site redesign wiki page](https://github.com/JATS4R/jats4r.github.io/wiki/Site-redesign).]

**Comment by seligym** [comment]
Sounds great to me. Yes, I will set up GA for the site.

And yes, let's not get bogged down in process; don't worry, as you've
probably gathered by now, I like to keep things moving :), and there are
ways of doing this so that people don't wind up feeling as though they want
to stab themselves ;-}

Let's add the idea of site renovation to the agenda at the next group
meeting, but just to ask whether everyone is OK with going down this road
(I don't want to take up too much time from the group's main work). Then I
agree, we could use this forum to communicate ideas/goals. I can even start
us rolling with some ideas, and then people can shoot them down or add or
tweak or whatever. Just so we're not starting with a totally blank canvas.

**Comment by Klortho** [comment]
Great!

**Comment by Melissa37** [comment]
Hi there

Great discussion and all good stuff. I have two things to add and it would be great to add this to the agenda of the next call, but I propose we do not spend longer that 15 in on it as we need to work on some next recommendations :-)

Defining goals
We've been using user stories a lot and find them really useful. Is it worth us having somewhere where people can add user stories as they think of them over the next couple of weeks, which will help you shape the requirement?

Sub-committee
I think this is a great idea, so those who are really interested in this can discuss and action, keeping things moving and agile, without having to consult on everything with the whole group - just reporting back on progress and decisions?
For instance as we are so open in our actions, GitHub issues could be labelled using the sub group name and then those involved need to read/act on them, and those not can either read/get involved or ignore depending on their time constraints or interest levels?

This is all fabulous stuff!

M

**Comment by seligym** [comment]
Hey Melissa

Agreed that we don't want to take up time with the main group meeting on
this. I think we were thinking that we would just float the idea to the
group, get approval to form a sub-committee (and ask for participation),
and start working on this in parallel with other JATS4R activities,
reporting at intervals or when a larger group decision really does need to
be made.

User stories are a great idea, and I use something similar when carrying
out information architecture research for any new site. So yes please add
user stories. This will help a lot.We need to capture user goals as well as
business goals for the site.

**Comment by Klortho** [comment]
I've updated the [site redesign wiki page](https://github.com/JATS4R/JATS4R-Participant-Hub/wiki/Site-redesign) with a few requirements that are pretty obvious, and I added a place for "user stories"

**Comment by Klortho** [comment]
Discussion now taking place on the JATS4R-admin list.
